fe_utils/finite_elements.py

```diff
@@ -144,9 +176,30 @@ def __init__(self, cell, degree):
         <ex-lagrange-element>`.
         """
 
-        raise NotImplementedError
+        # Provide the nodes of the equispaced Lagrange elements:
+        nodes = lagrange_points(cell, degree)
+
+        # Initialize entity_nodes dictionary:
+        if cell.dim == 1:
+            entities = [(0,0), (0,1), (1,0)]
+            entity_nodes = {0: {0: [], 1: []}, 1: {0: []}}
+        elif cell.dim == 2:
+            entities = [(0,0), (0,1), (0,2), (1,0), (1,1), (1,2), (2,0)]
+            entity_nodes =  {0: {0: [], 1: [], 2: []}, 1: {0: [], 1: [], 2: []}, 2: {0: []}}
```

Comment:
This information is already available as `cell.topology`, you don't need to write it out again.